### PR TITLE
Docs/release post v1.2

### PR DESCRIPTION
**Pull request by samajammin**
# Description

Blog post for upcoming MACI v1.2.0 release

## Related issue(s)

#856 

## Confirmation

- [x] I have read and understand MACI's [contributor guidelines](https://maci.pse.dev/docs/contributing) and [code of conduct](https://maci.pse.dev/docs/contributing/code-of-conduct).
- [x] I have read and understand MACI's [GitHub processes](https://github.com/privacy-scaling-explorations/maci/discussions/847).
- [x] I have read and understand MACI's [testing guide](https://maci.pse.dev/docs/testing).

### CODE REVIEW

File: website/blog/2024-02-28-maci-v1.2.0.md [hunk 1]
```diff
@@ -0,0 +1,140 @@
...
+This platform will serve as the definitive resource for all information related to MACI, including blog releases, documentation updates, and roadmap progress. Please report any inconsistencies you may find. As always, we welcome suggestions on how to make it better.
+
+## Security Audit
+
+Thanks to a thorough review by PSE's internal audit team and clr.fund's core developer, [yuetloo](https://github.com/yuetloo), we've identified and addressed several bugs during our recent refactoring efforts.
+
+Most notably, a critical bug in MACI v1.x discovered by [Kyle](https://github.com/kcharbo3), which could have allowed coordinators to censor votes, has been [fixed](https://github.com/privacy-scaling-explorations/maci/pull/1170).
+
+For more details on our recent audit, please refer to the following report:
+TODO ADD LINK TO AUDIT REPORT
```
Comment (samajammin) [inline website/blog/2024-02-28-maci-v1.2.0.md:106]: TODO ADD LINK TO AUDIT REPORT

File: website/blog/2024-02-28-maci-v1.2.0.md [hunk 2]
```diff
@@ -0,0 +1,140 @@
...
+tags: [release, audit, security, documentation]
+---
+
+# MACI v1.2 Release
+
+We are pleased to announce the release of MACI [v1.2.0](https://github.com/privacy-scaling-explorations/maci/releases/tag/v1.2.0)!
+
+This is our first release since [MACI v1.1.1](/blog/maci-v1-1-1-release) just over one year ago. This new release focuses on improved developer experience, security, performance and clearer documentation for users looking to learn and use MACI.
+
+## Background
```
Comment (kittybest) [inline website/blog/2024-02-28-maci-v1.2.0.md:18]: I know this is a draft PR, but thinking about this part is actually same as the one in v1.1.1, should we just remove it? I think the most important part in this v1.2.0 report is 1) which parts we refactored? 2) why do the refactors and 3) how they affect usage?

Comment (samajammin) [reply website/blog/2024-02-28-maci-v1.2.0.md:18]: Fair callout, I do think worth asking if this should be included in every release post or not. Maybe instead we should just reference the Introduction page for this type of background info?

Comment (samajammin) [reply website/blog/2024-02-28-maci-v1.2.0.md:18]: I'd say for now my (loosely held) opinion is to keep it in, given that we call out
- the quadratic & non-quadratic nature of it
- clr.fund is in the process of upgrading to v1.2

But I could be easily convinced to update both release posts to just reference this info on the Introduction page

File: website/blog/2024-02-28-maci-v1.2.0.md [hunk 3]
```diff
@@ -0,0 +1,121 @@
...
+
+### Flexible Voting Strategies
+
+After years of built-in quadratic voting (QV) in MACI, we now support non-quadratic voting polls. This opens up the door to potential integrations with a wider variety of projects, such as DAO governance applications.
+
+The new version of the Tally circuit (specific for non-QV), has reduced constraints, enabling a quicker proof generation process for vote tallying.
+
+We invite projects interested in leveraging this secure, on-chain voting mechanism to reach out for potential integrations. We hope to continue to expand support for additional voting methodologies, so please let us know your project's needs!
+
+### New Gatekeepers (EAS and Hats Protocol)
```
Comment (kittybest) [inline website/blog/2024-02-28-maci-v1.2.0.md:65]: I think the **Hats Protocol** is not yet merged?

Comment (ctrlc03) [reply website/blog/2024-02-28-maci-v1.2.0.md:65]: Good point! we could say it's in progress for now and include in 1.2.1?

**Pull request merged**